src/edit.js

```diff
@@ -25,6 +25,9 @@ import { InspectorControls } from '@wordpress/block-editor';
  * @return {WPElement} Element to render.
  */
 export default function Edit({ attributes, isSelected, setAttributes }) {
+	const blockProps = useBlockProps({
+		className: 'wp-block-bu-slideshow-slideshow-block'
```

Comment:
you might not need to specify the class name for the block here. Generally that is handled automatically. Unless this class name is different from the autogenerated classname assigned based on the block's name? It's also possible that might only work in conjunction with a block.json file... But thought I'd mention it in case you want to test removing it.